include/ccl_massfunc.h

```diff
@@ -22,8 +22,7 @@ void ccl_cosmology_compute_sigma(ccl_cosmology *cosmo, int *status);
  */
 void ccl_cosmology_compute_hmfparams(ccl_cosmology *cosmo, int *status);
 
-// TODO: smooth_mass is not really correct in this function, tho it makes sense in compute_sigma
```

Comment:
Did we address this todo or is it no longer relevant?

Reply:
ok, this was a naming issue. This appeared as `smooth_mass` in the `.h` and ass `halomass` in the `.c`. I've just homogenized to `halomass`.